### PR TITLE
HUE-9714 [Django Upgrade] [CommandParser] __init__() takes 1 position…

### PR DESCRIPTION
**Pull request by agl29**
…al argument but 2 were given

### CODE REVIEW

**Comment by agl29** [comment]
https://docs.djangoproject.com/en/2.1/_modules/django/core/management/base/ 
looks like `cmd` removed from the `__init__` function of `CommandParser` class. Please take a look. @romainr

File: desktop/core/src/desktop/manage_entry.py [hunk 1]
```diff
@@ -85,16 +85,16 @@ def entry():
 
   if len(sys.argv) > 1:
     subcommand = sys.argv[1]
-  parser = CommandParser(None, usage="%(prog)s subcommand [options] [args]", add_help=False)
+  parser = CommandParser(usage="%(prog)s subcommand [options] [args]", add_help=False)
```
Comment (romainr) [inline desktop/core/src/desktop/manage_entry.py:88]: Would it Django 1?

File: desktop/core/src/desktop/manage_entry.py [hunk 2]
```diff
@@ -85,16 +85,19 @@ def entry():
 
   if len(sys.argv) > 1:
     subcommand = sys.argv[1]
-  parser = CommandParser(None, usage="%(prog)s subcommand [options] [args]", add_help=False)
+  if sys.version_info[0] < 3:
+    parser = CommandParser(None, usage="%(prog)s subcommand [options] [args]", add_help=False)
+  else:
+    parser = CommandParser(usage="%(prog)s subcommand [options] [args]", add_help=False)
```
Comment (romainr) [inline desktop/core/src/desktop/manage_entry.py:91]: Any way to avoid the duplications?

e.g.
if py2:
  args = [None]
else:
  args = []
CommandParser(*args, **kwargs)

and even maybe
CommandParser(*args, usage="%(prog)s subcommand [options] [args]", add_help=False)

File: desktop/core/requirements.txt [hunk 3]
```diff
@@ -9,7 +9,7 @@ channels-redis==2.3.2
 configobj==5.0.6
 cryptography==3.2
 django-auth-ldap==1.3.0
-Django==2.0.13 # Django 2 then 3?
+Django==2.1.15 # Django 2 then 3?
```
Comment (romainr) [inline desktop/core/requirements.txt:12]: Should we do this on in a separate commit?

Comment (agl29) [reply desktop/core/requirements.txt]: yes, just push for checking that it is working in CI or not.

**Review by romainr — APPROVED** [review]
